doc/nrf/known_issues.rst

```diff
@@ -1206,7 +1213,7 @@ NCSDK-14117: Build fails for nRF52840DK in the ``prj_b0_wwcb`` configuration
 nRF5340 Audio
 =============
 
-.. rst-class:: v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
+.. rst-class:: v2-2-0 v2-1-2 v2-1-1 v2-1-0 v2-0-2 v2-0-1 v2-0-0
```

Comment:
Please add:
OCT-2347: In bidirectional mode, if a stream is running and the headset is reset: The gateway cannot reestablish the stream properly.

OCT-2070: Using USB-C to USB-C fails to enumerate on some Windows systems.

Reply:
Edited.